programs/dynamic-bonding-curve/src/instructions/creator/ix_withdraw_creator_migration_fee.rs

```diff
@@ -0,0 +1,92 @@
+use anchor_lang::prelude::*;
+use anchor_spl::token_interface::{Mint, TokenAccount, TokenInterface};
+
+use crate::{
+    const_pda,
+    state::{MigrationFeeDistribution, PoolConfig, VirtualPool, CREATOR_MASK},
+    token::transfer_from_pool,
+    EvtCreatorWithdrawMigrationFee, PoolError,
+};
+
+/// Accounts for creator withdraw migration fee
+#[event_cpi]
+#[derive(Accounts)]
+pub struct CreatorWithdrawMigrationFeeCtx<'info> {
+    /// CHECK: pool authority
+    #[account(
+        address = const_pda::pool_authority::ID
+    )]
+    pub pool_authority: UncheckedAccount<'info>,
+
+    #[account(has_one = quote_mint)]
+    pub config: AccountLoader<'info, PoolConfig>,
+
+    #[account(
+        mut,
+        has_one = quote_vault,
+        has_one = config,
+        has_one = creator,
+    )]
+    pub virtual_pool: AccountLoader<'info, VirtualPool>,
+
+    /// The receiver token account
+    #[account(mut)]
+    pub token_quote_account: Box<InterfaceAccount<'info, TokenAccount>>,
+
+    /// The vault token account for output token
+    #[account(mut, token::token_program = token_quote_program, token::mint = quote_mint)]
+    pub quote_vault: Box<InterfaceAccount<'info, TokenAccount>>,
+
+    /// The mint of quote token
+    pub quote_mint: Box<InterfaceAccount<'info, Mint>>,
+
+    pub creator: Signer<'info>,
+
+    /// Token b program
+    pub token_quote_program: Interface<'info, TokenInterface>,
+}
+
+pub fn handle_creator_withdraw_migration_fee(
```

Comment:
Why don't have a common function?
The only difference I saw was `Signer` and `mask` and emitted `event`

Reply:
Fixed